packages/common/src/PriceIdentifierUtils.js

```diff
@@ -36,6 +36,7 @@ const OPTIMISTIC_ORACLE_IGNORE_POST_EXPIRY = [
   "uSTONKS_APR21",
   "GASETH-TWAP-1Mx1M",
   "uSTONKS_JUN21",
+  "uTVL_KPI_UMA",
```

Comment:
So, this should be ignored post-expiry, but uses some other pricefeed pre-expiry, like a dex?

Reply:
Nope, should be ignored at all times. Switched lists